OpenRA.Mods.Common/Traits/HitShape.cs

```diff
@@ -74,6 +74,14 @@ public class HitShape : ConditionalTrait<HitShapeInfo>, ITargetablePositions
 		ITargetableCells targetableCells;
 		Turreted turret;
 
+		(
+			(CPos Cell, SubCell SubCell)[] targetableCells,
```

Comment:
Personally, I'd prefer
```
((CPos Cell, SubCell SubCell)[] targetableCells,
```
on the first line and a newline before `cachedTargetablePositions`, but maybe that's just me.

Reply:
Done